sql/core/src/main/scala/org/apache/spark/sql/internal/HiveSerDe.scala

```diff
@@ -72,7 +72,7 @@ object HiveSerDe {
   def sourceToSerDe(source: String): Option[HiveSerDe] = {
     val key = source.toLowerCase(Locale.ROOT) match {
       case s if s.startsWith("org.apache.spark.sql.parquet") => "parquet"
-      case s if s.startsWith("org.apache.spark.sql.orc") => "orc"
```

Comment:
We also need to keep the original one. We do not want to introduce behavior breaks.

Reply:
ok